test/test_sparse.py

```diff
@@ -1068,6 +1068,7 @@ def run_test(sizes):
         # more sophisticated algos
         run_test((10, 100, 100))
 
+    @onlyCPU
```

Comment:
Are you introducing this because otherwise the test would take too long?

Reply:
No, this explicitly tests parallel_for in the CPU kernel.